Readme.md

```diff
@@ -62,11 +62,6 @@ well put it to good use.
 - From the drop down, select `^ Control` under the `Caps Lock` setting
 - In the `Select Keyboard` drop down, set it for both internal and external keyboards
 
-### Fonts
-
-You look at text all damn day. Do yourself a favor and install the
-[MesloGM font][font] and rock the medium variant, size `12`.
-
```

Comment:
I added some notes in [this commit](https://github.com/ello/dotfiles/commit/bf89f7db0017f72f443f4941d152e4f7c4514c5f). Basically all Meslo did was increase the line-heights on Apple's Menlo font. You can now control this in your terminal (which the polarized.terminal file has the settings for) theme.